Automattic/jetpack
# Components: Add `logo` property to AdminPage component

This PR adds the logo` property to `<AdminPage />` component in order to be able to change the properties of the `<JetpackLogo />` rendered by the component. We need it for https://github.com/Automattic/jetpack/issues/23014#issuecomment-1049080541

Fixes #

#### Changes proposed in this Pull Request:

*

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Test with Storybook
* cd `./jetpack/projects/js-packages/storybook`
* `pnpm install & pnpm storybook:dev`
* Swtich `<PageAdmin />` story
* Using ..., confirm you can set the `height`, `showText`, and other Jetpack props.

<img width="976" alt="Screen Shot 2022-02-23 at 3 54 53 PM" src="https://user-images.githubusercontent.com/77539/155388129-6ed58bea-74c3-4ec2-aab8-6995fd69543f.png">
